Gifski/DropVideoViewController.swift

```diff
@@ -0,0 +1,113 @@
+import AppKit
+import Crashlytics
+import AVKit
+
+final class DropVideoViewController: NSViewController {
+	private lazy var videoDropView = with(VideoDropView()) {
+		$0.frame.size = CGSize(width: 360, height: 240)
+		$0.dropText = "Drop a Video to Convert to GIF"
+		$0.onComplete = { [weak self] url in
+			NSApp.activate(ignoringOtherApps: true)
+			self?.convert(url.first!)
```

Comment:
`url.first` reads weird. I think we should change the drop view to only return a single URL.